src/engraving/libmscore/fermata.h

```diff
@@ -44,7 +44,7 @@ class Fermata final : public EngravingItem
     OBJECT_ALLOCATOR(engraving, Fermata)
 
     SymId _symId;
-    double _timeStretch;
+    double _timeStretch = -1;
```

Comment:
Better initialise as `-1.0`

Reply:
Okay, just asking is there a specific reason for doing so. Or is it just for looks?

Reply:
Apart from that it is slightly clearer, it eliminates an implicit conversion from int to double.

Reply:
Oh okay